.gitignore

```diff
@@ -22,3 +21,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
```

Comment:
Why do you have .env change in the signup Pull Request ?
Remember A pull request should never be out of the scope of the Task